lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {
+  describe,
+  beforeEach,
+  afterEach,
+  it,
+} from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import { mountWithContext } from '../../../../test/bigtest/helpers';
+import {
+  ProfilesLabelInteractor,
+  ProfilesPopoverInteractor,
+} from './interactors';
+import ProfilesLabel from '../ProfilesLabel';
+
+import translations from '../../../../translations/stripes-data-transfer-components/en';
+
+describe('ProfilesLabel', () => {
+  const profilesLabel = new ProfilesLabelInteractor();
+  const popover = new ProfilesPopoverInteractor();
+
+  describe('rendering ProfileLabel', () => {
+    const link = 'http://localhost/';
+    const content = <div data-profiles-label-content>Content</div>;
+
+    beforeEach(async () => {
+      const overlayContainer = document.createElement('div');
+
+      overlayContainer.id = 'OverlayContainer';
+      document.body.append(overlayContainer);
+
+      await mountWithContext(
+        <ProfilesLabel
+          link={link}
+          content={content}
+        />
+      );
+    });
+
+    afterEach(() => {
+      document.getElementById('OverlayContainer').remove();
+    });
+
+    it('should display ProfileLabel component', () => {
+      expect(profilesLabel.isPresent).to.be.true;
+    });
+
+    it('should display the correct label text', () => {
+      expect(profilesLabel.text).to.equal(translations.profiles);
+    });
+
+    it('should display info icon button', () => {
+      expect(profilesLabel.infoButton.isPresent).to.be.true;
+    });
+
+    describe('clicking on info icon button', () => {
+      beforeEach(async () => {
+        await profilesLabel.infoButton.click();
+        await new Promise(resolve => { setTimeout(() => resolve(), 1000); });
```

Comment:
I think in the long run it makes sense to move this to utils with the ability to pass timer value as a parameter.

Reply:
Yes, you are right. Let's do it when we have other places with timeout.